modules/cms/src/frontend/views/commands/block/create_block.php

```diff
@@ -123,6 +123,18 @@ public function extraVars()
     */
     public function admin()
     {
-        return '<p><?= $name; ?> Admin View</p>';
+        return  '<p style="display: block; text-align: right; font-weight: bold; text-transform: uppercase;"><?= $name; ?> Admin View</p>'.
```

Comment:
'There was 1 failure:

1) luyatests\core\components\FormatterTest::testDefaultSettingsFromGivenArray
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-2017/2017/2017
+2017201720172017/2017201720172017/2017201720172017

/Users/silvan/Sites/luya_dev/luya/tests/core/components/FormatterTest.php:22
/Users/silvan/Sites/luya_dev/luya/vendor/phpunit/phpunit/phpunit:52

FAILURES!
Tests: 204, Assertions: 631, Failures: 1.'